dev-docs/bidders/connectad.md

```diff
@@ -1,17 +1,22 @@
 ---
 layout: bidder
 title: ConnectAd
-description: Prebid Serverbid Bidder Adaptor
+description: ConnectAd Prebid Adaptor
 hide: true
 biddercode: connectad
-aliasCode: serverbid
+media_types: banner
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+userIds: pubcommonId, unifiedId/tradedesk, digitrustId, id5Id, liveIntent, parrable
```

Comment:
to stay consistent with other adapters, can you please modify this line to the following?

```
userIds: digitrust, id5Id, liveIntentId, parrableId, pubCommonId, unifiedId
```

Reply:
Hi, sure thing!